http_test.go

```diff
@@ -109,12 +105,17 @@ func Test_EndToEnd(t *testing.T) {
 		})
 
 		Convey("Handling error conditions", func() {
-			Convey("When no page is specified", func() {
```

Comment:
This test needed to be changed because now we want to return document metadata when no page is specified.